Makefile

```diff
@@ -186,6 +186,7 @@ manifest-gen-test:
 
 shellcheck:
 	# --external-sources because .buildkite/scripts/common/get-test-artifacts.sh source .env
+        @[[ ! -f .env ]] && touch .env || true
```

Comment:
CI reports:
```
Makefile:189: *** missing separator (did you mean TAB instead of 8 spaces?).  Stop.
```

Reply:
@thbkrkr I was using `git apply /path/to/diff`, and clearly it didn't interpret spaces vs tabs well... commit pushed to fix.